phygnn/model_interfaces/phygnn_model.py

```diff
@@ -146,15 +149,15 @@ def train_model(self, features, labels, p, n_batch=16, n_epoch=10,
             Number of times to iterate on the training data.
         shuffle : bool
             Flag to randomly subset the validation data and batch selection
-            from features and labels.
+            from x and y.
```

Comment:
I'm confused by the use of features/labels vs x/y

Reply:
i just use x and y as kind of shorthand in the phygnn model while the model interfaces prefer the features and labels var names. I just keep copying over doc strings so thats why this is getting messed up. will fix.